tools/bin/deploy_docusaurus

```diff
@@ -79,6 +79,12 @@ if ! yarn build; then
   exit 1
 fi
 
+# "check PR mode" -- we run "yarn build" and stop before "yarn deploy"
+if [ "${SKIP_DEPLOY}" = "yes" ]; then
```

Comment:
SKIP_DEPLOY doesn't exist for local runs

Reply:
which is perfectly fine for local deploys

Reply:
we get then `if [ "" = "yes" ]; then ... "